listenbrainz/webserver/static/js/src/utils.tsx

```diff
@@ -1,42 +1,40 @@
-/* eslint-disable */
-// TODO: Make the code ESLint compliant
-// TODO: Port to typescript
-
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import React from "react"; // jsx compiled to React.createElement
+import * as React from "react";
+import _ from "lodash";
 import { faPlayCircle } from "@fortawesome/free-solid-svg-icons";
 
-export function getSpotifyEmbedUriFromListen(listen) {
+import { IconProp } from "@fortawesome/fontawesome-svg-core"; // eslint-disable-line no-unused-vars
```

Comment:
We'll need to look at this. IconProp is a type that is used in the file later. However, the no-unused-vars rule didn't pick it up. I think it's a compatibility issue between typescript and the rule.